test/committee/utils.js

```diff
@@ -39,20 +34,39 @@ function shuffleAccount(dislocation, account) {
 }
 
 function distance(h1, h2) {
-  // create BN from hashes
-  var a = new BN(h1, 16);
-  var b = new BN(h2, 16);
-  // return XOR as big number
+  // Create BN from hashes.
+  const a = new BN(h1, 16);
+  const b = new BN(h2, 16);
+
+  // Return XOR as big number.
   return a.xor(b);
 }
 
-module.exports = {
+function distanceToProposal(dislocation, account, proposal) {
+  return distance(shuffleAccount(dislocation, account), proposal);
+}
 
-  createCommittee,
+const CommitteeStatus = {
+  Open: new BN(0),
+  Cooldown: new BN(1),
+  CommitPhase: new BN(2),
+  RevealPhase: new BN(3),
+  Closed: new BN(4),
+  Invalid: new BN(5),
+};
 
-  shuffleAccount,
+function isCommitteeOpen(status) {
+  return CommitteeStatus.Open.cmp(status) === 0;
+}
 
-  distance,
+const SENTINEL_MEMBERS = '0x1';
```

Comment:
should this not be 20 bytes?

Reply:
Yes. However, as it was not used (currently, we retrieve sentinel value from contract itself) I've removed it. Thx.